### ISSUE TITLE
ScopeIndent can fail when an opening parenthesis is on a line by itself

### ISSUE THREAD

**Issue by oligriffiths** [squizlabs/PHP_CodeSniffer#704]
Run this sniff over the following:

```php
$a =
    (
        $b = 1
    );
$c = 2;
```

You get the error `7 | ERROR | [x] Line indented incorrectly; expected at least 4 spaces, found 0`

This is correctly indented (and an abbreviated example). The code for this sniff is quite complex which is why I've not submitted a PR.

Any ideas?

**Comment by gsherwood** [comment]
I think I figured it out. Thanks for reporting.

**Comment by oligriffiths** [comment]
Anytime. 

Got a couple of others that I'll submit PRs for too.
